lib/git/config.go

```diff
@@ -15,6 +15,13 @@ import (
 	"github.com/Originate/git-town/lib/util"
 )
 
+// AddPerennialBranch adds the given branch as a perennial branch
+func AddPerennialBranch(branchName string) {
+	oldValue := GetPerennialBranches()
+	newValue := append(oldValue, branchName)
```

Comment:
There are too many temp variables here, which is a code smell. Also, `oldValue` doesn't describe much. I would inline `oldValue` and possibly also `newValue`. 

```go
setPerennialBranches(append(GetPerennialBranches(), branchName))